pom.xml

```diff
@@ -93,7 +93,7 @@
         <dependency>
             <groupId>org.ohdsi.sql</groupId>
             <artifactId>SqlRender</artifactId>
-            <version>1.7.0-SNAPSHOT</version>
+            <version>1.19.4</version>
```

Comment:
I think the problem is we build this from this https://github.com/SevKohler/Eos/blob/f57603ba8d92b386e226733282829724c83db65d/pom.xml#L136

Do we still need this, since it seems the SQLrenderer is also on Maven ? 
https://mvnrepository.com/artifact/org.ohdsi.sql/SqlRender

I would recommend we just delete this nexus thing  @prabash what do you think ?
Would also resolve future problems with the build sine we had this already one time.